CannibalisticZombies/Assets/Scripts/Character/Player/BulletSpawnPoint.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class BulletSpawn : MonoBehaviour
```

Comment:
What is the difference between the scripts `BulletSpawnPoint `and `PlayerCombatController`?

Also note that the class name must be the same name as the script, otherwise Unity won't recognize it.